gtp2ogs.js

```diff
@@ -1154,6 +1195,10 @@ class Connection {
                 this.processMove(gamedata);
             } */
 
+	    // Don't connect to old finished games.
```

Comment:
I'd been meaning to eventually change this myself. :D Since it fills my logs with connect/disconnect on already finished games.